Kontent.Ai.Management.Tests/Kontent.Ai.Management.Tests.csproj

```diff
@@ -33,6 +33,25 @@
 		<ProjectReference Include="..\Kontent.Ai.Management\Kontent.Ai.Management.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+		<AssemblyAttribute Include="Kontent.Ai.Management.Attributes.SourceTrackingHeader">
```

Comment:
I might add a comment linking to the actual test.

Reply:
It's a test project and if you remove this attribute one test will fail, isn't a sufficient link?